components/_share.scss

```diff
@@ -0,0 +1,170 @@
+.sharing-info {
+  clear: both;
+  margin-top: 50px;
+
+  @include span-columns(8 of 12);
+  @include shift(2);
+
+  h2 {
+    margin-bottom: $base-font-size;
+  }
+}
+
+.sharing-box {
+  clear: both;
+  border: 2px solid #F0F0F1;
+  margin: auto;
+  padding: 20px;
+  background-color: #FFFFFF;
+
+  @include media($desktop) {
+    width: 465px;
+  }
+
+  .sharing-preview {
+    border: 1px solid #FCE0D7;
```

Comment:
Color literals like `#FCE0D7` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FCE0D7` should be written as `#fce0d7`<br>Properties should be ordered -webkit-animation-duration, -webkit-animation-duration, animation-duration, animation-duration, -webkit-animation-fill-mode, animation-fill-mode, border, border-radius